src/popup/router/components/TokenAmount.vue

```diff
@@ -19,11 +19,12 @@ export default {
   props: {
     amount: { type: Number, required: true },
     symbol: { type: String, default: 'AE' },
-    altText: { type: String },
+    altText: { type: String, default: '' },
     hideFiat: { type: Boolean },
     direction: {
       type: String,
       validator: (value) => ['sent', 'received'].includes(value),
+      default: 'sent',
```

Comment:
Using this approach we will have `-amount` with red color where it's not needed, for example in invite links:
![image](https://user-images.githubusercontent.com/7098449/118923345-be05c880-b97e-11eb-9b0e-2acbb22443ef.png)
Let's maybe introduce empty direction which will looks neutral
![image](https://user-images.githubusercontent.com/7098449/118923432-e097e180-b97e-11eb-9571-6632c931375e.png)

```suggestion
      validator: (value) => ['sent', 'received', ''].includes(value),
      default: '',
```

Reply:
prefer `default: undefined` in this case instead of changing validator